app/controllers/two_factor_authentication/otp_expired_controller.rb

```diff
@@ -22,5 +24,17 @@ def redirect_unless_otp_expired_redirect_enabled
     def otp_delivery_preference
       current_user.otp_delivery_preference
     end
+
+    def next_path
+      if mfa_context.enabled_mfa_methods_count < 1
+        authentication_methods_setup_url
+      else
+        login_two_factor_options_url
+      end
+    end
+
+    def use_another_phone_number
+      mfa_context.phone_configurations.none? || mfa_context.enabled_mfa_methods_count <= 1
```

Comment:
Couple comments:

- Conventionally, I'd expect a method which returns a boolean to end with `?` and be named to communicate what's being returned.
- On that note, I'm not really clear what's being returned here. Are we trying to limit the display of the option for only the cases where the user is adding a new phone number?
   - Maybe we could do something similar to how we use [`unconfirmed_phone?`](https://github.com/18F/identity-idp/blob/6183c2e7d7594b47d57293b833cc748c90d762bf/app/controllers/concerns/two_factor_authenticatable_methods.rb#L292-L306) to control this on the previous screen, changing this method to something like...

```suggestion
    def unconfirmed_phone?
      user_session[:unconfirmed_phone].present? && UserSessionContext.confirmation_context?(context)
```

Reply:
This method was intended for users that are setting up their phone number on sign-up and when a user is adding a new phone number after the account exists. 

I can remove the check from adding a new phone number and see if that is what we want because that piece was added based on a gut check.